src/Microsoft.CodeAnalysis.Razor.Workspaces/RazorCompletionFactsService.cs

```diff
@@ -4,7 +4,7 @@
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Language;
 
-namespace Microsoft.VisualStudio.Editor.Razor
+namespace Microsoft.CodeAnalysis.Razor
 {
     public abstract class RazorCompletionFactsService
```

Comment:
public? You said this was internal earlier.

Reply:
Haven't shipped tooling with these yet (did you in your dev16 primer?), these are brand new APIs that I just added not too long ago. Anyhow, nothing depends on the public API of this and I have IVT in other places. Will make em internal.